src/extension.ts

```diff
@@ -27,6 +31,15 @@ export async function activate(context: vscode.ExtensionContext) {
   registerShowcaseModule(context);
 
   const keysForSync = ["followedUsers", "repos"].map((key) => `gistpad.${key}`);
+  if (config.get("tracing.enableOutputChannel")) {
+    tracing = new Trace();
+  }
+
+  tracing?.appendLine(
```

Comment:
Is this message really useful? Since it's not dependent on any user state or action, it feels like we could omit it. Otherwise, it's just logging a static list.

Reply:
It helped me debug the sync problem (I used it to compare what the list looked like in globalStorage on different machines at startup); maybe we could really implement different levels of tracing and log I mentioned in a different comment? If you think it's overkill I can remove it, no problem